wormhole-connect/src/views/Bridge/TransferLimitedWarning.tsx

```diff
@@ -1,25 +1,219 @@
 import React from 'react';
 import useIsTransferLimited from 'hooks/useIsTransferLimited';
-import AlertBanner from 'components/AlertBanner';
 import { wh } from 'utils/sdk';
+import { Dialog, ScopedCssBaseline } from '@mui/material';
+import CloseIcon from 'icons/Close';
+import Button from 'components/Button';
+import ClockIcon from 'icons/Clock';
+import { makeStyles } from 'tss-react/mui';
+
+const useStyles = makeStyles()((theme: any) => ({
+  dialog: {
+    zIndex: 10,
+  },
+  paper: {
+    borderRadius: 24,
+    [theme.breakpoints.down('sm')]: {
+      maxWidth: '100%',
+      maxHeight: '100%',
+      margin: 0,
+      borderRadius: 0,
+    },
+  },
+  container: {
+    position: 'relative',
+    width: '100%',
+    margin: '0 auto',
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  modal: {
+    width: '100%',
+    padding: 32,
+    position: 'relative',
+    maxHeight: 'calc( 100vh - 80px )',
+    [theme.breakpoints.down('sm')]: {
+      padding: '24px 12px',
+      maxHeight: '100%',
+    },
+  },
+  close: {
+    color: '#3C3D77',
+    position: 'absolute',
+    top: '28px',
+    right: '28px',
+    cursor: 'pointer',
+    opacity: '70%',
+    zIndex: '10',
+    [theme.breakpoints.down('sm')]: {
+      top: '12px',
+      right: '20px',
+    },
+  },
+  background: {
+    background: theme.palette.modal.background,
+  },
+  icon: {
+    textAlign: 'center',
+    paddingTop: 24,
+    paddingBottom: 24,
+  },
+  title: {
+    textAlign: 'center',
+    fontSize: 24,
+    fontWeight: 600,
+    lineHeight: '32px',
+    paddingTop: 24,
+    paddingBottom: 24,
+    borderBottom: '1px solid #313266',
+  },
+  text: {
+    fontSize: '16px',
+    fontWeight: 400,
+    lineHeight: '24px',
+    color: '#9495D6',
+    paddingTop: 24,
+    paddingBottom: 24,
+  },
+  link: {
+    color: '#D5D5EB',
+    textDecorationLine: 'underline',
+  },
+  orange: {
+    color: '#E48329',
+  },
+  clock: {
+    width: 64,
+    height: 64,
+  },
+  buttonContainer: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    marginTop: '24px',
+    gap: 10,
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+    },
+  },
+}));
+
+const learnMoreLink =
+  'https://github.com/wormhole-foundation/wormhole/blob/main/whitepapers/0007_governor.md';
 
 const TransferLimitedWarning = () => {
+  const { classes } = useStyles();
+  const [open, setOpen] = React.useState(true);
```

Comment:
I think we might want to set `open` to `true` when the `chain` or the `token` changes. A user might close the dialog (which sets `open` to `false`) and then select a different chain and/or token which is also limited, but the dialog won't show for this new token.

Reply:
You are right, done!